Extension Project_Final.Rmd

```diff
@@ -0,0 +1,768 @@
+---
+title: "Extension of Villarreal Paper"
+author: "Reisman and Lajous"
+date: "April 23, 2015"
+output: html_document
+---
+
+We chose to replicate Andrés Villarreal's 2004 AJS paper, "The Social Ecology of Rural Violence: Land Scarcity, the Organization of Agricultural Production, and the Presence of the State." Applying an ecological theory of crime to the study of rural violence in Mexico, Villarreal attempts explain the fact that rural municipalities have high variation in homicide rates by investigating the effects of land distribution, collective ownership, and the organization of agricultural production on homicide rates. He tests five specific hypotheses: (1) When agricultural land is scarce relative to the number of individuals, there will be more conflict and therefore more homicides; (2) An unequal distribution of land will lead to more violent conflict; (3) When property rights are not well enforced or are contingent, there will be more conflict over lands. For this reason, ejido and communal units will experience more violence; (4) agricultural production systems that involve more commodified relations of production and exchange will be associated with a breakdown of community social cohesion and therefore with more violence, and the introduction of cash crops will lead to greater conflict over resources and control over trade, and consequently to more violence; and (5) remote mountainous areas will have higher rates of violence due to the absence of state institutions.
+
+We were unable to replicate Villarreal's findings, due in part to changes in Mexican data disclosure laws. For our extension of Villarreal's paper, we decided to update his analysis with newer (2007) data -- his original analysis was conducted using 1991 data. In doing so, we decided to remove the variables for which we identified data issues (The Thiel Index, Percent of Plots 5 Hectares or Less, and Log of Average Plot Size), as well as one variable for which 2007 data is not available (Percent Subsistence Agriculture). We correspondingly decided to include several new variables in the analysis, including a different indicator of land inequality to replace the Thiel index (the Gini Index), a new indicator of state presence not used by Villarreal (Number of Policemen per 1,000 people) and a new control variable -- the Human Development Index (HDI). We hope that these changes and additions will produce a valid analysis that we can compare to Villrreal's findings.
+
```

Comment:
sounds good.  What was the results of the comparison?  Were your findings different?